src/tabs.scss

```diff
@@ -23,11 +44,21 @@
 
       &:focus-visible {
         outline: none;
+
+        .react-tabs__tab-inner {
+          border: 3px solid $rpf-brand-raspberry;
+        }
       }
   
       &:focus:after {
         display: none;
-       }
+      }
+
+      &:first-of-type {
+        .react-tabs__tab-inner {
+          margin: 8px 4px 8px 8px;
+        }
+      }
```

Comment:
Is there a way of putting this extra margin on the right of last of type as well for consistency (esp when there are enough tabs to enable scrolling)

Reply:
Yup! Done in [26e35d2](https://github.com/RaspberryPiFoundation/editor-ui/pull/299/commits/26e35d20e04528eb1e75c991d9a99dc3db3bbee5)